docs/iam-usage.rst

```diff
@@ -0,0 +1,148 @@
+Using IAM For Users
+===================
+
+.. note:: This document assumes basic knowledge of Google Cloud IAM,
+   see `the docs <https://cloud.google.com/iam/docs/>`_ for details
+
+Resources that implement the IAM interface provide the following methods:
+
+Policy Editing Convenience Methods
+----------------------------------
+
+Common keyword arguments:
+
+- ``retries`` number of times to retry in the
```

Comment:
This effectively adds a feature to the library that should be more general than IAM.